.github/workflows/pr.yml

```diff
@@ -21,9 +21,15 @@ jobs:
             - name: Setup repo
               uses: ./.github/actions/setup
 
+            - name: Typecheck
```

Comment:
We shouldn't need this here since the setup won't pass unless at least the package itself type checks successfully.

The rest (e.g. the website) I think makes sense to handle when we actually need to use it.

Reply:
Does the build script also typecheck all files? I wasn't sure.